### ISSUE TITLE
FirebaseRecaptchaVerifierModal not loading in web browser view

### ISSUE THREAD

**Issue by teeaaspoon** [expo/expo#11101]
### Summary of Issue 
Trying to implement phone auth in my expo project. It works fine in ios and android however when I try to do web browser view FirebaseRecaptchaVerifierModal does not load at all

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
 Expo CLI 3.28.5 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 3.2.57 - /bin/bash
    SDKs:
      iOS SDK:
        Platforms: iOS 14.2, DriverKit 20.0, macOS 11.0, tvOS 14.2, watchOS 7.1
    IDEs:
      Xcode: 12.2/12B45b - /usr/bin/xcodebuild
    npmPackages:
      expo: ~39.0.2 => 39.0.4 
      react: 16.13.1 => 16.13.1 
      react-dom: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-39.0.4.tar.gz => 0.63.2 
      react-native-web: ~0.13.12 => 0.13.18 
    npmGlobalPackages:
      expo-cli: 3.28.5
    Expo Workflow: managed

### Reproducible Demo

web view recaptcha will not load like below 
![image](https://user-images.githubusercontent.com/16376909/99891320-aa2f1300-2c36-11eb-96cb-8b6a289235ee.png)

### Steps to Reproduce
Basically followed this guide https://medium.com/@arjayosma/set-up-firebase-phone-authentication-in-expo-sdk-37-without-ejecting-8a472460b1cf

**Comment by AdamJNavarro** [comment]
Hey @teeaaspoon, could you please provide an runnable repro? We won't be able to help debug this issue with screenshots and links to tutorials. We need to see your actual implementation/code to better understand the issue and identify potential issues/fixes.

Cheers,
Adam

**Comment by IjzerenHein** [comment]
Hi! This had been added to `expo-firebase-recaptcha@1.4.0`. This will be released with SDK 41 but you can already use it by installing `expo-firebase-recaptcha@next`.

**Comment by Delord-Nzanzu** [comment]
hello, I am between test in react native authentication with firebase, all seems to work well, no error, I can not have recaptcha, here is the image. thank you for helping me.
![Imag](https://user-images.githubusercontent.com/63587598/124162267-9be38800-da9e-11eb-8f49-2420081310b2.PNG)

**Comment by Vindi-Van** [comment]
was this resolved? I am having the same issue.
